pygmt/datasets/samples.py

```diff
@@ -123,3 +123,26 @@ def load_fractures_compilation():
     fname = which("@fractures_06.txt", download="c")
     data = pd.read_csv(fname, header=None, sep=r"\s+", names=["azimuth", "length"])
     return data[["length", "azimuth"]]
+
+
+def load_hotspots():
+    """
+    Load a table with the locations, names, and suggested symbol sizes of
+    hotspots.
+
+    This is the ``@hotspots.txt`` dataset used in the GMT tutorials.
+
+    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
+    first time you invoke this function. Afterwards, it will load the data from
+    the cache. So you'll need an internet connection the first time around.
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        The data table with columns "longitude", "latitude", "symbol_size", and
+        "placename".
```

Comment:
The name styles of "symbol_size" and "placename" are inconsistent. We should use "symbol_size" and "place_name" or "symbolsize" and "placename".

Reply:
I prefer the underscore, so it's now "place_name"